src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -89,11 +89,17 @@ private function fixIsset(Tokens $tokens, int $index): void
             return; // some weird stuff inside the isset
         }
 
+        $issetCode = strtolower($issetTokens->generateCode());
+
+        if ('$this' !== $issetCode) {
+            return; // null coalescing operator does not with $this
+        }
+
         // search what is inside the middle argument of ternary operator
         $ternaryColonIndex = $tokens->getNextTokenOfKind($ternaryQuestionMarkIndex, [':']);
         $ternaryFirstOperandTokens = $this->getMeaningfulSequence($tokens, $ternaryQuestionMarkIndex, $ternaryColonIndex);
 
-        if ($issetTokens->generateCode() !== $ternaryFirstOperandTokens->generateCode()) {
+        if ($issetCode !== $ternaryFirstOperandTokens->generateCode()) {
```

Comment:
```suggestion
        if ($issetCode !== strtolower($ternaryFirstOperandTokens->generateCode())) {
```
Shouldn't we also be case-insensitive here?

Reply:
For the record - [here's the reasoning why not](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7052#issuecomment-1589047018).

Reply:
Yep, and now it will also be guaranteed to be case-sensitive by unit tests 👌